usrtools/txgen/app/cli-functions.c

```diff
@@ -184,31 +188,26 @@ set_cmd(int argc, char **argv)
             break;
         case 30:
             p = strchr(argv[4], '/');
-            if (!p) {
-                cne_printf("src IP address should contain subnet value, default /32 for IPv4, /128 for IPv6\n");
-            }
-            ip_ver = _atoip(argv[4],
-                CNE_IPADDR_V4 | CNE_IPADDR_NETWORK,
-                &ip, sizeof(ip));
-            if(ip_ver > 0) {
-                foreach_port(portlist,
-                    single_set_ipaddr(info, 's', &ip, IPv4_VERSION));
-            } else
-                cne_printf("ERROR couldn't set ipv4IP address\n");
+            if (!p)
+                CNE_ERR_RET("src IP address should contain subnet value, default /32 for IPv4, /128 for IPv6\n");
+            *p++ = '\0';
+            prefixlen = atoi(p);
```

Comment:
the result of "atoi" needs to be validated.

Reply:
Converted to a strtol() call instead as it returns an if found.